tools/lsp/completion.rs

```diff
@@ -87,14 +87,14 @@ pub(crate) fn completion_at(
             // add keywords
             r.extend(
                 [
-                    ("property", "property <$1> $2;"),
-                    ("in property", "in property <$1> $2;"),
-                    ("in-out property", "in-out property <$1> $2;"),
-                    ("out property", "out property <$1> $2;"),
-                    ("private property", "private property <$1> $2;"),
-                    ("function", "function $1() {}"),
-                    ("public function", "public function $1() {}"),
-                    ("callback", "callback $1();"),
+                    ("property", "property <${1:int}> ${2:name};"),
+                    (
+                        "qualified property",
```

Comment:
That is wrong IMHO.   This is for editor that don't support snippets.  It should not complete to `qualified property`  because that's not valid.
Also why would we want to auto-complete that with `qual<tab>` ?
But we want to auto complete  `in<tab>`  or just `i<tab>`

Please just keep 4 entries there for all possibilities.

Reply:
> The editor needs to support snippets for any of this to work. 

If the editor doesn't support snippet, it will just use the label instead of the snippet. And the "qualifié property" is the label, so it d use that.

Anyway, thanks for changing